addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/handler/RFXComHandler.java

```diff
@@ -165,101 +166,105 @@ public void onDeviceMessageReceived(ThingUID bridge, RFXComMessage message) {
+                                        break;
+                                    case RAW_MESSAGE:
+                                        updateState(CHANNEL_RAW_MESSAGE, message.convertToState(valueSelector));
+                                        break;
+                                    case RAW_PAYLOAD:
+                                        updateState(CHANNEL_RAW_PAYLOAD, message.convertToState(valueSelector));
+                                        break;
+                                    case SET_POINT:
+                                        updateState(CHANNEL_SET_POINT, message.convertToState(valueSelector));
+                                        break;
+                                    case SHUTTER:
+                                        updateState(CHANNEL_SHUTTER, message.convertToState(valueSelector));
+                                        break;
+                                    case SIGNAL_LEVEL:
+                                        updateState(CHANNEL_SIGNAL_LEVEL,
+                                                convertSignalLevelToSystemWideLevel(message.convertToState(valueSelector)));
+                                        break;
+                                    case STATUS:
+                                        updateState(CHANNEL_STATUS, message.convertToState(valueSelector));
+                                        break;
+                                    case TEMPERATURE:
+                                        updateState(CHANNEL_TEMPERATURE, message.convertToState(valueSelector));
+                                        break;
+                                    case TOTAL_AMP_HOUR:
+                                        updateState(CHANNEL_TOTAL_AMP_HOUR, message.convertToState(valueSelector));
+                                        break;
+                                    case TOTAL_USAGE:
+                                        updateState(CHANNEL_TOTAL_USAGE, message.convertToState(valueSelector));
+                                        break;
+                                    case VOLTAGE:
+                                        updateState(CHANNEL_VOLTAGE, message.convertToState(valueSelector));
+                                        break;
+                                    case WIND_DIRECTION:
+                                        updateState(CHANNEL_WIND_DIRECTION, message.convertToState(valueSelector));
+                                        break;
+                                    case WIND_SPEED:
+                                        updateState(CHANNEL_WIND_SPEED, message.convertToState(valueSelector));
+                                        break;
+                                    default:
+                                        logger.debug("Unsupported value selector '{}'", valueSelector);
+                                        break;
+                                }
```

Comment:
We should only catch here some specific exceptions, with the current setup we convert each exception to a debug statement. I don't prefer real bugs to be hidden like that, can we just catch the exceptions which are reported when unsupported messages are received.